x-pack/plugins/enterprise_search/server/routes/enterprise_search/index.ts

```diff
@@ -9,6 +9,7 @@ import { RouteDependencies } from '../../plugin';
 
 import { registerDocumentRoute } from './documents';
 import { registerEnginesRoutes } from './engines';
+import { registerEnginesSearchRoutes } from './engines_search';
```

Comment:
Looks like this is breaking CI. We need to either rename back to `engines_search` or update this to `engines_preview`